src/Microsoft.Azure.WebJobs.Host/Tables/CollectorArgumentBindingProvider.cs

```diff
@@ -22,8 +23,8 @@ public IStorageTableArgumentBinding TryCreate(ParameterInfo parameter)
             }
 
             Type entityType = GetCollectorItemType(parameter.ParameterType);
-
-            if (!TableClient.ImplementsOrEqualsITableEntity(entityType))
+                        
+            if (!TableClient.ImplementsOrEqualsITableEntity(entityType) && !TypeUtility.IsJObject(entityType))
             {
```

Comment:
Can you file a bug with a SDK example? 

---
In reply to: [85970127](https://github.com/Azure/azure-webjobs-sdk/pull/891#discussion_r85970127) [](ancestors = 85970127)